web/profiles/joinup/joinup.behat.inc

```diff
@@ -1701,6 +1701,15 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
   }
 
   /**
+   * Wait until the id="updateprogress" element is gone, or until timeout.
```

Comment:
Hehe this documentation is only useful for people that know what this ID stands for :)